src/dataflow/api.rs

```diff
@@ -38,8 +38,12 @@ pub struct OperatorBuilder<OP: Operator, Backend = DefaultBackend> {
 }
 
 impl<OP: Operator, Backend: arcon_state::Backend> OperatorBuilder<OP, Backend> {
-    pub(crate) fn create_backend(&self, state_dir: std::path::PathBuf) -> Arc<Backend> {
-        Arc::new(Backend::create(&state_dir).unwrap())
+    pub(crate) fn create_backend(
+        &self,
+        state_dir: std::path::PathBuf,
+        name: &'static str,
```

Comment:
Can be changed back to String as Backend now takes it as an argument.

```suggestion
        name: String
```